roles/nfs-server/templates/exports.j2

```diff
@@ -1,4 +1,9 @@
 # {{ ansible_managed }}
 # fsid=X is needed for XFS filesystems with 64bit inodes
-/srv/{{ nfs_server }}/veyepar {{ eth_local_ip_address|default(ansible_default_ipv4.network + '/' + ansible_default_ipv4.netmask) }}(ro,async,no_subtree_check,fsid=0)
-/srv/{{ nfs_server }}/video {{ eth_local_ip_address|default(ansible_default_ipv4.network + '/' + ansible_default_ipv4.netmask) }}(rw,sync,no_subtree_check,fsid=1)
+{% if eth_local_ip_address is defined %}
```

Comment:
I was trying to duplicate what the old code was doing:
{{ eth_local_ip_address|default(...

which made me wonder what the point was.